library/Mockery/Matcher/MatcherInterface.php

```diff
@@ -10,14 +12,16 @@
 
 namespace Mockery\Matcher;
 
-interface Matcher
+interface MatcherInterface
 {
     /**
      * Check if the actual value matches the expected.
      * Actual passed by reference to preserve reference trail (where applicable)
      * back to the original method parameter.
      *
-     * @param mixed $actual
+     * @template TActual
```

Comment:
This template type is not useful in any way. It's the same as just mixed @ghostwriter

Reply:
This template type is to used by static analysis tools like psalm, to infer the exact `type` of the argument that is being passed in.

Reply:
Yes I'm aware :) But in this case you've only used it for one of the parameters, meaning it's actual type is inferred on every call but isn't used anywhere, so it's as good as `@param mixed $actual` in this case, no different.

Reply:
You're right, it only used in this case, dropped the template tag.

thanks for your feedback.